{"uuid": "db92a657-c88f-4773-9faf-89b96449dcb7", "children": ["4dab7194-5f06-422f-b0a6-9f056816e9fd"], "befores": [{"name": "classSetup", "status": "broken", "statusDetails": {"message": "AttributeError: 'TestLogin' object has no attribute 'driver'\n", "trace": "  File \"C:\\Users\\ADMIN\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\site-packages\\pluggy\\_callers.py\", line 167, in _multicall\n    teardown.throw(outcome._exception)\n  File \"C:\\Users\\ADMIN\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\site-packages\\_pytest\\setuponly.py\", line 36, in pytest_fixture_setup\n    return (yield)\n            ^^^^^\n  File \"C:\\Users\\ADMIN\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\site-packages\\pluggy\\_callers.py\", line 103, in _multicall\n    res = hook_impl.function(*args)\n          ^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\ADMIN\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\site-packages\\_pytest\\fixtures.py\", line 1125, in pytest_fixture_setup\n    result = call_fixture_func(fixturefunc, request, kwargs)\n             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^\n  File \"C:\\Users\\ADMIN\\AppData\\Local\\Programs\\Python\\Python311\\Lib\\site-packages\\_pytest\\fixtures.py\", line 887, in call_fixture_func\n    fixture_result = fixturefunc(**kwargs)\n                     ^^^^^^^^^^^^^^^^^^^^^\n  File \"F:\\AGEST\\Selenium\\SeleniumPython\\tests\\test_login.py\", line 17, in classSetup\n    self.login_page = LoginPage(self.driver)\n                                ^^^^^^^^^^^\n"}, "start": 1721026310878, "stop": 1721026310878}], "start": 1721026310878, "stop": 1721026310894}